pkg/util/tls_config.go

```diff
@@ -67,6 +67,12 @@ func NewTLSConfigWithPassword(clientCert, clientKey, clientKeyPassword, caCert s
 	return config, nil
 }
 
+// SetCACertDirs sets location(s) containing CA certificates which should be trusted for
+// all future calls to CreateTLSClientConfig
+func SetCACertDirs(caCertDirs []string) {
+	setCACertDirs(caCertDirs)
+}
```

Comment:
Does it make sense having the function here just to call to the inner function? Maybe we can just remove this one an export the other one. WDYT?